crates/ruff_benchmark/benches/ty.rs

```diff
@@ -610,7 +617,7 @@ fn datetype(criterion: &mut Criterion) {
             max_dep_date: "2025-07-04",
             python_version: PythonVersion::PY313,
         },
-        0,
+        2,
```

Comment:
What are the two new diagnostics here? Are they legit?

Reply:
Yeah it uses two deprecated time apis, same ones that the ecosystem checks flagged on a ton of packages.